packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -210,28 +212,47 @@ export class RemoteMediaGroup {
     }
   }
 
-  private sendActiveSpeakerMediaRequest(commit: boolean) {
+  public sendActiveSpeakerMediaRequest(commit: boolean) {
     this.cancelActiveSpeakerMediaRequest(false);
 
-    this.mediaRequestId = this.mediaRequestManager.addRequest(
-      {
-        policyInfo: {
-          policy: 'active-speaker',
-          priority: this.priority,
-          crossPriorityDuplication: false,
-          crossPolicyDuplication: false,
-          preferLiveVideo: !!this.options?.preferLiveVideo,
+    if (this.options.namedMediaGroup?.value) {
```

Comment:
why do we need a whole big if and else block for namedMediaGroup? it's just 1 extra parameter, the rest of the request remains the same right? so we could instead just add 1 line:
`namedMediaGroups: this.options.namedMediaGroup?.value ? [this.options?.namedMediaGroup] : undefined,`
right?

Reply:
Previously I used if else block for considering the case that there may be multi groups , but in current implementation, yes, we can simplify it